swagger_server/controllers/helper.py

```diff
@@ -1,8 +1,69 @@
-def is_valid_file_format(file_name, extensions):
+import base64
+
+
+def is_valid_file_extension(file_name, extensions):
+    """
+    Check the file format is valid
+    :param file_name: The file name to be checked
+    :param extensions: The list of extensions that are valid
+    :return: boolean
+    """
     extensions_tuple = tuple([e for e in extensions.split(",")])
```

Comment:
Forgot about this one. The ons.env.get is returning an unusable string "xls,xlsx" which natural wouldn't work, so I put this in to get around it. I will have another look